lib/util/plist.js

```diff
@@ -0,0 +1,56 @@
+import bplistParser from 'bplist-parser';
```

Comment:
Is there any crossover of these functions and the ones in `appium-support`?

https://github.com/appium/appium-support/blob/master/lib/plist.js

Reply:
Kinda. I can move these slowly there. It would be nicer if they are centralised there

Reply:
Yeah. Anything that is available now through `appium-support` ought to be used. Other things can, eventually, be moved. But perhaps leaving them until it is known that they are exactly what is needed is best. 

It might be a good idea to noodle about in `appium-support`, as well, just to familiarize yourself with the utilities that are there, so you don't reinvent the wheel.